### ISSUE TITLE
Seasonal Shop page is blank

### ISSUE THREAD

**Issue by lemoness** [HabitRPG/habitica#9449]
Currently on Safari, the Seasonal Shop does not display, so only the Footer is showing. This is not an issue on Firefox.

<img width="1417" alt="screen shot 2017-11-08 at 6 39 25 pm" src="https://user-images.githubusercontent.com/3494081/32585979-00aea624-c4b5-11e7-8e9d-2157baa62343.png">

**Comment by majestic-owl448** [comment]
The same issue on Google Chrome!

**Comment by SabreCat** [comment]
The following appears in the JS console:

```
TypeError: Cannot read property 'items' of undefined
    at a.seasonal (11.962348407626515e688f.js:1)
    at wi.get (vendor.7ae272c2c1386c8b8bff.js:6)
    at wi.evaluate (vendor.7ae272c2c1386c8b8bff.js:6)
    at a.seasonal (vendor.7ae272c2c1386c8b8bff.js:6)
    at a.n (11.962348407626515e688f.js:1)
    at a.t._render (vendor.7ae272c2c1386c8b8bff.js:6)
    at a.r (vendor.7ae272c2c1386c8b8bff.js:6)
    at wi.get (vendor.7ae272c2c1386c8b8bff.js:6)
    at new wi (vendor.7ae272c2c1386c8b8bff.js:6)
    at At (vendor.7ae272c2c1386c8b8bff.js:6)
```

**Comment by Tressley** [comment]
Is this still an issue for anyone? I'm having trouble replicating in Chrome and Safari this morning.

**Comment by majestic-owl448** [comment]
On Chrome 62.0.3202.89, Windows 10, the Seasonal Shop is not visible.

Il lunedì 13 novembre 2017, Tressley Cahill <notifications@github.com> ha
scritto:

**Comment by lemoness** [comment]
fixed in c8ca67a

**Comment by negue** [comment]
I think we can close this, the seasonal shop isn't blank anymore.
